demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3961 @@
+{
```

Comment:
<p>Now, the nodes. Again, we have to choose the columns' names. The names of the 0/1-columns don't matter so much, but we can give the first column (of IDs) and the last one (of subjects) useful names.</p><p> </p><p>(added apostrophe after columns; doesn't -&gt; don't; avoided saying "useful" twice)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>